### ISSUE TITLE
For help: "zh_Hant_TW" and "zh_Hans_CH"

### ISSUE THREAD

**Issue by LinHaoLove** [bdlukaa/fluent_ui#529]
In my project, you need to use "en", "zh_Hant_TW" and "zh_Hans_CH".
But I haven't found a way to make them exist at the same time. Can anyone help me? Thank you.

I saw this in the official document. Does it mean that traditional Chinese cannot be used?

```dart
const List<Locale> defaultSupportedLocales = <Locale>[
  Locale('ar'),
  Locale('de'),
  Locale('en'),
  Locale('es'),
  Locale('fr'),
  Locale('hi'),
  Locale('pt'),
  Locale('ru'),
  Locale('zh'),
];
```

```
The following is the information printed out by the console.

`======== Exception caught by widgets ===============================================================
The following message was thrown:
Warning: This application's locale, zh_Hans_CH, is not supported by all of its localization delegates.

• A FluentLocalizations delegate that supports the zh_Hans_CH locale was not found.

The declared supported locales for this app are: en, zh_Hans_CH, zh_Hant_TW

See https://flutter.dev/tutorials/internationalization/ for more information about configuring an app's locale, supportedLocales, and localizationsDelegates parameters.
====================================================================================================

======== Exception caught by widgets ===============================================================
The following message was thrown:
Warning: This application's locale, zh_Hans_CH, is not supported by all of its localization delegates.

• A FluentLocalizations delegate that supports the zh_Hans_CH locale was not found.

The declared supported locales for this app are: en, zh_Hans_CH, zh_Hant_TW

See https://flutter.dev/tutorials/internationalization/ for more information about configuring an app's locale, supportedLocales, and localizationsDelegates parameters.
====================================================================================================
``` 

**Edit WinXaito: formatting**

**Comment by WinXaito** [comment]
Which version of fluent_ui do you use ? 

Apparently, this is fix on the master branch (for the 4.0.0): https://github.com/bdlukaa/fluent_ui/pull/459

 Can you check with the branch master ?

**Comment by LinHaoLove** [comment]
@WinXaito 
Hi,Sir
I provided the following information:

fluent_ui: ^3.12.0 （The latest stable version）
intl: ^0.17.0
The "Flutter intl"（Latest version） From Android Studio Markerplace.
Flutter version 3.3.1 on channel stable.

I have tried to pre release version 4.0.0,it is ok.
Thank you very much for your reply.
